UPGRADE.md

```diff
@@ -0,0 +1,35 @@
+
+  - Core middleware functionality -> [`@middy/core`](https://www.npmjs.com/package/@middy/core)
+  - `cache` -> [`@middy/cache`](https://www.npmjs.com/package/@middy/cache)
+  - `cors` -> [`@middy/http-cors`](https://www.npmjs.com/package/@middy/http-cors)
+  - `doNotWaitForEmptyEventLoop` -> [`@middy/do-not-wait-for-empty-event-loop`](https://www.npmjs.com/package/@middy/do-not-wait-for-empty-event-loop)
+  - `httpContentNegotiation` ->  [`@middy/http-content-negotiation`](https://www.npmjs.com/package/@middy/http-content-negotiation)
+  - `httpErrorHandler` ->  [`@middy/http-error-handler`](https://www.npmjs.com/package/@middy/http-error-handler)
+  - `httpEventNormalizer` ->  [`@middy/http-event-normalizer`](https://www.npmjs.com/package/@middy/http-event-normalizer)
+  - `httpHeaderNormalizer` ->  [`@middy/http-header-normalizer`](https://www.npmjs.com/package/@middy/http-header-normalizer)
+  - `httpMultipartBodyParser` ->  [`@middy/http-json-body-parser`](https://www.npmjs.com/package/@middy/http-json-body-parser)
+  - `httpPartialResponse` ->  [`@middy/http-partial-response`](https://www.npmjs.com/package/@middy/http-partial-response)
+  - `jsonBodyParser` ->  [`@middy/http-json-body-parser`](https://www.npmjs.com/package/@middy/http-json-body-parser)
+  - `s3KeyNormalizer` ->  [`@middy/s3-key-normalizer`](https://www.npmjs.com/package/@middy/s3-key-normalizer)
+  - `secretsManager` ->  [`@middy/secrets-manager`](https://www.npmjs.com/package/@middy/secrets-manager)
+  - `ssm` ->  [`@middy/ssm`](https://www.npmjs.com/package/@middy/ssm)
+  - `validator` ->  [`@middy/validator`](https://www.npmjs.com/package/@middy/validator)
+  - `urlEncodeBodyParser` ->  [`@middy/http-urlencode-body-parser`](https://www.npmjs.com/package/@middy/http-urlencode-body-parser)
+  - `warmup` ->  [`@middy/warmup`](https://www.npmjs.com/package/@middy/warmup)
+
+
+## Header normalization in `http-header-normalizer`
+
+In Middy 0.x the `httpHeaderNormalizer` middleware normalizes HTTP header names into their own canonical format, for instance `Sec-WebSocket-Key` (notice the casing). In Middy 1.0 this behavior has been changed to provide header names in lowercase format (e.g. `sec-webSocket-key`). This new behavior is more consistent with what Node.js core `http` package does and what other famous http frameworks like Express or Fastify do, so this is considered a more intuitive approach.
+When updating to Middy 1.0, make sure you double check all your references to HTTP headers and switch to the lowercase version to read them.
+All the middy core modules have been already updated to support the new format, so you should worry only about your userland code.
+
+
+## Node.js 10 and 12 now supported / Node.js 6 and 8 now dropped
+
+Version 1.0 of Middy does not support Node.js versions 6.x and 8.x. These versions are now out of support, so you are highly encourage to move to Node.js 12 or 10, which are the new supported versions in Middy 1.0.
```

Comment:
`does not support` -> `no longer supports`
Just a subtle change

Might also be worth mentioning that AWS lambda has dropped support for these versions as well.